src/client.js

```diff
@@ -231,10 +238,14 @@ module.exports = class Client {
     if (this._body == null) {
       return null
     } else if (typeof this._body === 'object') {
-      if (this._isJsonApi) {
-        return JSON.stringify(recursive(this._body, kebabCase))
+      if (this._disableCase) {
```

Comment:
And then this small change here

```suggestion
      if (this._bodyCasing) {
```